sources/install/package_network.sh

```diff
@@ -164,6 +164,19 @@ function install_chisel() {
     add-to-list "chisel,https://github.com/jpillora/chisel,Go based TCP tunnel with authentication and encryption support"
 }
 
+function install_penelope() {
+    colorecho "Installing Penelope"
+    git -C /opt/tools clone --depth 1 https://github.com/brightio/penelope.git
+    cd /opt/tools/penelope || exit
+    python3 -m venv --system-site-packages ./venv   
+    source ./venv/bin/activate  
+    deactivate
```

Comment:
The project has a [pyproject.toml](https://github.com/brightio/penelope/blob/main/pyproject.toml), so you can use `pipx` to install it, which will be cleaner.